### PR TITLE
update action

### PR DESCRIPTION
**Pull request by smicheleaws**
update manual workflow with correct naming

### CODE REVIEW

File: .github/workflows/build-deploy.yml [hunk 1]
```diff
@@ -70,7 +70,8 @@ jobs:
           --namespace ${{ vars.KUBE_NAMESPACE }} \
           -f helm-chart/environments/values-${{ inputs.env }}.yaml \
           --set image.tag=${{ github.sha }} \
-          --set image.repository=${{ steps.login-ecr.outputs.registry }}/${{ vars.DOCKER_IMAGE_NAME}} \
-          --set serviceAccount.name=${{ vars.DOCKER_IMAGE_NAME}} \
+          --set image.repository=${{ steps.login-ecr.outputs.registry }}/${{ vars.DOCKER_IMAGE_NAME }} \
+          --set serviceAccount.name=${{ vars.DOCKER_IMAGE_NAME }} \
           --set serviceAccount.namespace=${{ vars.KUBE_NAMESPACE }} \
```
Comment (emanuelr93) [inline .github/workflows/build-deploy.yml]: Please use variable in values-dev.yaml for Namespace

File: .github/workflows/build-deploy.yml [hunk 2]
```diff
@@ -70,7 +70,8 @@ jobs:
...
-          --set ingress.hosts[0].paths[0].path=/${{ vars.DOCKER_IMAGE_NAME}}/
+          --set ingress.hosts[0].paths[0].path=/${{ vars.DOCKER_IMAGE_NAME }}/ \
+          --set "ingress.annotations.alb\.ingress\.kubernetes\.io/load-balancer-name=${{ vars.ALB }}"
```
Comment (emanuelr93) [inline .github/workflows/build-deploy.yml]: Please use variable on dev file for ALB name.

**Review by emanuelr93 — CHANGES_REQUESTED** [review]

**Comment by smicheleaws** [comment]
variables updated

**Review by emanuelr93 — DISMISSED** [review]
LGTM

**Review by emanuelr93 — APPROVED** [review]

**Pull request merged**